inference-engine/include/vpu/myriad_plugin_config.hpp

```diff
@@ -54,8 +54,6 @@ DECLARE_VPU_MYRIAD_CONFIG_KEY(PLATFORM);
  * @brief Supported keys definition for VPU_MYRIAD_CONFIG_KEY(PLATFORM) option.
  */
 INFERENCE_ENGINE_DEPRECATED("")
-DECLARE_VPU_MYRIAD_CONFIG_VALUE(2450);
-INFERENCE_ENGINE_DEPRECATED("")
 DECLARE_VPU_MYRIAD_CONFIG_VALUE(2480);
```

Comment:
Why has it remained? Do we need `DECLARE_VPU_MYRIAD_CONFIG_KEY(PLATFORM);` config if it can take only one value?

Reply:
Right, will be removed. Thanks